python/environment.yml

```diff
@@ -22,4 +22,4 @@ dependencies:
   # Extra
   - ipython
   - pysolar
-  - scipy
+  - monet
```

Comment:
This is when it flipped, since MONET installs xESMF, which leads to an netCDF Fortran getting installed in the conda env, which gives it a `nf-config` that isn't the one we want for the build.

Reply:
For now I'll try to add an env var for it.

Reply:
Seems to have worked